feature/stn/instrument/index.js

```diff
@@ -27,6 +30,8 @@ var PUSH_STATE = 'pushState'
 
 // Turn on feature harvesting
 var loader = require('loader')
+loader = Object.assign({}, loader, {features: Object.assign(loader.features || {}, {stn: true})})
+console.log('stn loader')
```

Comment:
remove console log

Reply:
whoops forgot to remove both of these lines before commiting.